specs/ScreenCaptureRequested.md

```diff
@@ -0,0 +1,357 @@
+            // CoreWebView2Frame.ScreenCaptureRequested exists, and both exist together,
+            // so it would not be a problem.
+            args.Handled = true;
+        };
+    }
+    catch (NotImplementedException exception)
+    {
+        // If the runtime support is not there we probably want this
+        // to be a no-op.
+    }
+};
+```
+
+# API Details
+## C++
+```
+interface ICoreWebView2_20;
+interface ICoreWebView2ScreenCaptureRequestedEventArgs;
+interface ICoreWebView2ScreenCaptureRequestedEventHandler;
+
+interface ICoreWebView2Frame3;
+interface ICoreWebView2FrameScreenCaptureRequestedEventHandler;
+interface ICoreWebView2FrameScreenCaptureRequestedEventArgs;
+
+/// This interface is an extension of `ICoreWebView2` that supports the ScreenCaptureRequested event. 
+// MSOWNERS: stevenwei@microsoft.com 
+[uuid(accc0e97-fa2d-4a8d-ad61-cc9ae57a1825), object, pointer_default(unique)] 
+interface ICoreWebView2_20 : IUnknown { 
+  /// Add an event handler for the `ScreenCaptureRequested` event. 
+  /// `ScreenCaptureRequested` event is raised when the Screen Capture API is requested by the user using getDisplayMedia(). 
+  HRESULT add_ScreenCaptureRequested( 
+      [in] ICoreWebView2ScreenCaptureRequestedEventHandler* eventHandler, 
+      [out] EventRegistrationToken* token); 
+  /// Remove an event handler previously added with `add_ScreenCaptureRequested`. 
+  // MSOWNERS: stevenwei@microsoft.com 
+  HRESULT remove_ScreenCaptureRequested( 
+      [in] EventRegistrationToken token); 
+}   
+/// Receives `ScreenCaptureRequested` events. 
+// MSOWNERS: stevenwei@microsoft.com 
+[uuid(9b5bbea1-4a58-4567-8b42-8781d3986cb4), object, pointer_default(unique)] 
+interface ICoreWebView2ScreenCaptureRequestedEventHandler : IUnknown { 
+  /// Called to provide the event args when a screen capture is requested on 
+  /// a WebView element. 
+  HRESULT Invoke( 
+      [in] ICoreWebView2* sender, 
+      [in] ICoreWebView2ScreenCaptureRequestedEventArgs* args); 
+} 
+/// Event args for the `ScreenCaptureRequested` event. 
+// MSOWNERS: stevenwei@microsoft.com 
+[uuid(a1d309ee-c03f-11eb-8529-0242ac130003), object, pointer_default(unique)] 
+interface ICoreWebView2ScreenCaptureRequestedEventArgs : IUnknown { 
+  /// The associated frame information that requests the screen capture  
+  /// permission. This can be used to grab the frame source, name, frameId,  
+  /// and parent frame information. 
+  [propget] HRESULT OriginalSourceFrameInfo([out, retval] ICoreWebView2FrameInfo**  
+     frameInfo); 
+  /// By default, both the `ScreenCaptureRequested` event handlers on the 
+  /// `CoreWebView2Frame` and the `CoreWebView2` will be invoked, with the 
+  /// `CoreWebView2Frame` event handlers invoked first. The host may 
+  /// set this flag to `TRUE` within the `CoreWebView2Frame` event handlers 
+  /// to prevent the remaining `CoreWebView2` event handlers from being  
+  /// invoked. 

```

Comment:
Also the remaining _frame_ handlers from being invoked?

Reply:
The text is currently correct. It will need to be updated in the future when we support grand+child frames.